2333/2/versus.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'libs/options_parser.rb'
+require_relative 'libs/handler.rb'
+require_relative 'libs/analyzer.rb'
+
+parameters = ParametersParser.new.options
+rappers = Handler.create_rappers_array
+
+if parameters[:top_bad_words]
+  versus_analyzer = Analyzer.new(rappers, parameters[:top_bad_words])
+  versus_analyzer.top_bad_words
+elsif parameters[:name]
+  put_count = parameters[:top_words] || 30
+  versus_analyzer = Analyzer.new(rappers, put_count, parameters[:name])
```

Comment:
I'm not 100% sure, it's a question for you. What do you think about making 2 separate analyzers for those 2 different jobs: 1) showing top bad words and 2) showing the top words for the specific rapper?

I think it would be better because those analyzers may have a different set of private methods. Also, two different analyzers may be initialized with different sets of arguments.